Model/LoggerHandlerSorter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Vaimo Group. All rights reserved.
+ * See LICENSE_VAIMO.txt for license details.
+ */
+
+namespace Vaimo\StdoutJsonLogger\Model;
+
+class LoggerHandlerSorter
+{
+    const STDOUT_HANDLER_POSITION = 1;
+
+    private bool $isApplied = false;
```

Comment:
We should probably run this through phpcs (and phpmd) with Magento settings; it'll warn about lacking docblock here